kagiso_smart_404/views.py

```diff
@@ -1,13 +1,18 @@
+from django.http import HttpResponsePermanentRedirect
 from django.shortcuts import render
 
-from .utils import suggest_page_from_misspelled_slug
+from .utils import determine_if_slug_matches_one_page_exactly, suggest_page_from_misspelled_slug
 
 
 def not_found(request):  # pragma: no cover
     slug = request.path
     root_page = request.site.root_page.specific
 
-    suggested_page = suggest_page_from_misspelled_slug(slug, root_page)
+    exact_match = determine_if_slug_matches_one_page_exactly(slug, root_page)
```

Comment:
this whole thing is basically pointless , if the existing method to suggest similar pages returns 1 item then redirect